axis_backend/apps/kpis/views/kpi_assignment_viewset.py

```
@@ -0,0 +1,171 @@
+
+        if not value:
+            return Response(
+                {'error': 'Measurement value is required'},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+        try:
+            assignment = self.service.record_measurement(pk, value, period)
+            serializer = self.detail_serializer_class(assignment)
+            return Response(serializer.data)
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)